docs/contributing.md

```diff
@@ -59,10 +59,12 @@ Guidelines](https://opensource.google/conduct/).
 
 ## Setting up a development environment
 
-To develop `jj`, the mandatory steps are simply to [install
-Rust](https://www.rust-lang.org/tools/install) (the default installer options
-are fine), clone the repository, and use `cargo build`, `cargo fmt`,
-`cargo test`, etc.  If you are preparing a PR, there are some additional
+To develop `jj`, the mandatory steps are simply
+to [install Rust](https://www.rust-lang.org/tools/install) (the default
+installer options are fine), clone the repository, and use `cargo build`
+, `cargo fmt`,
+`cargo clippy --workspace`, and  
```

Comment:
I'm not sure exactly what the difference is, but should --all-targets be replaced by --workspace in the `cargo watch` command below?

I'll test both out, but maybe not right this second.

Reply:
I think `--all-targets` makes it also run on tests, benches, and anything else, while `--workspace` makes it run on all crates in the workspace. So I should update to specify both, I think. Confusingly, `cargo fmt` runs on all targets and on the whole workspace by default, it seems. I've updated to pass both flags.

Reply:
👍